packages/core/src/Stores/portfolio-store.js

```diff
@@ -398,7 +398,7 @@ export default class PortfolioStore extends BaseStore {
 
     @action.bound
     onHoverPosition(is_over, position) {
-        const { symbol: underlying } = this.root_store.active_symbols;
+        const { symbol: underlying } = JSON.parse(localStorage.getItem('trade_store')) || {};
```

Comment:
Get symbol from updated localStorage symbols

Reply:
@mahdiyeh-fs Can we pass the symbol as a function param instead of getting it from the `localStorage`?

Reply:
@balakrishna-binary sure, will do that :pray: